packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -42,7 +42,7 @@ export function createFile(assembleAttachment: media.AssembleAttachment[], mimeT
  * @internal
  * Limited to Microsoft-internal use
  */
-export function decodeAttachment(attachment: media.MediaChunk, mimeType: string): media.AssembleAttachment {
+export function decodeAttachment(attachment: media.MediaChunk, mimeType: string): media.AssembleAttachment | null {
```

Comment:
>null [](http://example.com/codeflow?start=109&length=4)

Inconsistently, this function was already using `null` in the same way the above was using `undefined`. Now they are consistent in their use of `null` to intentionally indicate `no value here.`